summingbird-storm/src/main/scala/com/twitter/summingbird/storm/StormPlatform.scala

```diff
@@ -204,49 +202,58 @@ abstract class Storm(options: Map[String, Options], transformConfig: Summingbird
     }
     val metrics = getOrElse(stormDag, node, DEFAULT_SPOUT_STORM_METRICS)
 
-    val registerAllMetrics = Externalizer({ context: TopologyContext =>
+    val summerOpt: Option[SummerNode[Storm]] = stormDag.dependantsOf(node.asInstanceOf[StormNode]).collect { case s: SummerNode[Storm] => s }.headOption
+    val stormSpout = summerOpt match {
+      case Some(s) => {
```

Comment:
Can we call it summer instead of s? While the context for the block is small the surrounding context is pretty big so little verbosity seems useful here.